### PR TITLE
UI test setup

### PR DESCRIPTION
**Pull request by aryue123321**

### CODE REVIEW

File: test/uiTesting/mainPage.test.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { expect } from "chai";
+const puppeteer = require("puppeteer");
+
+const baseUrl = "https://tbd.mattmerr.com";
```
Comment (MatthewMerrill) [inline test/uiTesting/mainPage.test.js]: We'll want to make this localhost or something, right now it's opening browser to tbd.mattmerr.com. I'm not sure how we can launch a jekyll server while in the test though. Maybe exec "bundle exec jekyll serve" with an unusual port?

File: test/uiTesting/mainPage.test.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+  it("should back to main page", async () => {
+    await page.click("#navbar > ul > li:nth-child(1) > a");
+    const title = await page.title();
+    expect(title).to.contain("Statistic");
+  });
+
+  it("should display one mean page", async () => {
+    await page.waitForSelector("#main-menu > ul > li:nth-child(2) > a");
+    await page.click("#main-menu > ul > li:nth-child(2) > a");
```
Comment (MatthewMerrill) [inline test/uiTesting/mainPage.test.js]: I've pushed some barebones two-proportion, one-mean is now `nth-child(3)`
```suggestion
    await page.click("#main-menu > ul > li:nth-child(3) > a");
```

File: test/uiTesting/mainPage.test.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  });
+
+  it("should back to main page", async () => {
+    await page.click("#navbar > ul > li:nth-child(1) > a");
+    const title = await page.title();
+    expect(title).to.contain("Statistic");
+  });
+
+  it("should display one mean page", async () => {
+    await page.waitForSelector("#main-menu > ul > li:nth-child(2) > a");
```
Comment (MatthewMerrill) [inline test/uiTesting/mainPage.test.js]:
```suggestion
    await page.waitForSelector("#main-menu > ul > li:nth-child(3) > a");
```

**Review by MatthewMerrill — APPROVED** [review]
See comments, otherwise LGTM

File: test/uiTesting/mainPage.test.js (hunk 1 above)
Comment (aryue123321) [reply test/uiTesting/mainPage.test.js]: http://127.0.0.1:4000 is the default localhost address of jekyll
  to run this test, have to open another terminal to lanch the distrition to localhost
 the command is `jekyll serve`

**Pull request merged**